idd/Energy+.idd.in

```diff
@@ -63383,12 +63383,15 @@ Boiler:Steam,
        \key OtherFuel1
        \key OtherFuel2
   N1 , \field Maximum Operating Pressure
-       \units Kpa
+       \required-field
+       \units Pa
   N2 , \field Theoretical Efficiency
+       \required-field
```

Comment:
@mitchute Quick look at the code says this is initialized to zero, and then it's use here without any checks for a valid value.  

Took 5ZoneSteamBaseboard and set the steam boiler efficiency to zero and it runs and tallies heating gas at 1.8e308.  The minimum here should at least be >0, not 0.  I would go with default of 0.8 unless @rraustad can suggest something different.

Code throws an error if Design Outlet Steam Temeperature is zero.  Will run with 0.1.
Code also throws an error if Minimum Part Load Ratio is zero - even though IDD say minimum is zero.  I don't know why it shouldn't allow zero.